libfly/binding.cc

```diff
@@ -591,6 +698,12 @@ extern "C"
     // }
   }
 
+  bool js_run_module(const js_runtime *rt, js_compiled_module module_data) 
```

Comment:
This isn't getting called, so I'm assuming that's what you mean by "Move execution from v8 Script to Module." in #38?